irr/include/aabbox3d.h

```diff
@@ -20,9 +20,7 @@ template <class T>
 class aabbox3d
 {
 public:
-	//! Default Constructor.
-	constexpr aabbox3d() :
-			MinEdge(-1, -1, -1), MaxEdge(1, 1, 1) {}
+	constexpr aabbox3d() {}
```

Comment:
I would probably suggest getting rid of this constructor entirely (I think exe_virus also suggested that on Discord?) because the "origin-initialization" is also not much more useful and just a potential source of errors.

Practically the user should always be forced to decide what to initialize this to (ideally the first position if it's easily obtainable).

Reply:
What I can do is say `= default`. Since there are other constructor signatures, it won't generate a default one. I forgot about that, but the compiler reminded me with some error messages when I tried to compile without it.

Reply:
Ah, you are suggesting not permitting default initialization. Yeah, we can do that.

Reply:
FWIW `VoxelArea` solves this correctly by being empty (that means `min=(0,0,0) max=(-1,-1,-1)`) by default and if you e.g. add a point it will correctly recognize that the points are meant to be replaced